content/rs/administering/designing-production/networking/using-oss-cluster-api.md

```diff
@@ -8,6 +8,9 @@ categories: ["RS"]
 {{%excerpt-include filename="rs/concepts/data-access/oss-cluster-api.md" %}}
 For more information, see [Redis OSS Cluster API]({{< relref "/rs/concepts/data-access/oss-cluster-api.md" >}}).
 
+These multi-key commands are not allowed with OSS cluster API:
+DEL, MSET, MGET, EXISTS, UNLINK, TOUCH
```

Comment:
I probably didn't explain myself properly: these commands can be used in OSS cluster mode, but all keys in use are restricted to the same hash slot, which is the default behavior of OSS cluster.
This is unlike the default behavior of RE, which allows them to be in different hash slots.